src/selectors/entities/channels.js

```diff
@@ -371,6 +371,22 @@ export const getOtherChannels: (GlobalState) => Array<Channel> = createSelector(
     }
 );
 
+export const getOtherUnArchivedChannels: (GlobalState) => Array<Channel> = createSelector(
```

Comment:
do we actually need this selector?

Reply:
not needed. i've modified the original selector